galaxy/api/views/namespace.py

```diff
@@ -258,7 +258,7 @@ def update(self, request, *args, **kwargs):
                 owners.append(request.user.pk)
             update_owners(instance, owners)
 
-        for item in ('name', 'description', 'avatar_url', 'location', 'company', 'email', 'html_url', 'active'):
+        for item in ('name', 'description', 'avatar_url', 'location', 'company', 'email', 'html_url', 'active', 'partner'):
```

Comment:
I think 'partner' needs to be added above, on line 215, in the 'post' method of NamespaceList. It wouldn't let me add a comment to that line directly, but take a look.

Reply:
Added.